mix.exs

```diff
@@ -12,7 +12,7 @@ defmodule Timex.Mixfile do
       package: package(),
       deps: deps(),
       docs: docs(),
-      compilers: [:gettext] ++ Mix.compilers(),
+      compilers: Mix.compilers(),
```

Comment:
I think this line could be removed, as `Mix.compilers()` seems to be the default value if nothing is specified, given that I understand [the documentation](https://hexdocs.pm/mix/Mix.html) correctly.

Reply:
I think better to check Elixir version first Mix.compilers(), since removal of mix task compiler only apply to version 1.11+.

```suggestion
      compilers: compilers_opts(),
      
   defp compilers_opts() do
     case Version.compare(System.version(), "1.11.0") do
        :lt -> [:gettext] ++ Mix.compilers()
        _ -> Mix.compilers()
      end
   end
```

Reply:
@prihandi Thanks for your comment. Removing the `:gettext` compiler is a fix for _Gettext_ deprecating its usage, not _Elixir_ issuing a deprecation warning. Your suggested code change therefore should not be applied as it does not fix any backward compatibility issues.

In case I'm missing something, let me know!